LLMonFHIR/FHIR Display/FHIRResourcesView.swift

```diff
@@ -46,14 +70,49 @@ struct FHIRResourcesView: View {
                     }
                 }
                 .toolbar {
+                    ToolbarItem(placement: .navigationBarTrailing) {
+                        Button(action: {
+                            if !interpretingMultipleResources {
+                                showMultipleResourcesChat.toggle()
+                            } else {
+                            }
+                        }) {
+                            if interpretingMultipleResources {
+                                ProgressView("Interpreting...")
+                            } else {
+                                Text("Chat with All Records")
```

Comment:
Please use localized strings using the `Localizable.strings` file when you add new strings to the application.